x11-terms/alacritty/alacritty-0.12.3-r1.ebuild

```diff
@@ -303,6 +303,7 @@ src_unpack() {
 }
 
 src_configure() {
+	rust_pkg_setup
```

Comment:
It should be called in `pkg_setup` instead. Please also fix the commit message.

Reply:
I'd assume all other ebuilds would need it as well, since they are all look the same?

Reply:
Yes.

Reply:
I checked versions 0.13.2-r1 0.14.0-r1 there is no such error.
Previously there was a dependency, but it was removed by automerge. 

[history](https://github.com/gentoo/gentoo/commits/573e1bd41566e28cf85076c347fe2cf735961789/x11-terms/alacritty/alacritty-0.12.3-r1.ebuild)

Reply:
Ah, other versions don't need it because they don't inherit `python-any-r1` (so it's called for us).

You need to define one just for the older version with:
```
pkg_setup() {
	python-any-r1_pkg_setup
	rust_pkg_setup
}
```